src/main/webapp/WEB-INF/themes/root/app/error.ftl

```diff
@@ -41,10 +41,12 @@
 
     <p>Thank you for your patience.</p>
 
-    <div id="stackTrace">
-      <h2>Technical Information for Developers</h2>
-      <pre>${stackTrace?html}</pre>
-    </div>
+
+    <#if environment != "prod">
```

Comment:
@anilop I see stackTrace is also presented in these files.  I assume that they're all in use and should get the same treatment? 

src/main/webapp/WEB-INF/themes/desktop/ftl/error/error.ftl 
src/main/webapp/WEB-INF/themes/mobile/ftl/error/error.ftl
src/main/webapp/WEB-INF/themes/root/ftl/error/error.ftl

Also, I see a reference in:
plos-themes/code/desktop/all_sites/ftl/error/error.ftl

Reply:
Never mind about the reference in plos-themes, @tpolischuk covered it here: https://github.com/PLOS/plos-themes/pull/906

Reply:
@sgatchell - Agreed, they should get the same treatment.